cli/cmd/plugin/ui/web/tanzu-ui/src/views/providers/aws/AwsResources.reducer.tsx

```diff
@@ -0,0 +1,39 @@
+import { AwsResourceAction } from '../../../shared/types/types';
+import { ReducerDescriptor } from '../../../shared/utilities/Reducer.utils';
+import { AWS_ADD_RESOURCES, AWS_DELETE_RESOURCES } from '../../../state-management/actions/Resources.actions';
+
+export const STORE_SECTION_AWS_RESOURCES = 'resources';
+
+// The resources reducer builds up state by ensuring an object associated with action.datacenter,
```

Comment:
This is a great explanatory comment block!
However... I think `datacenter` is only associated with vSphere. Here it might make sense to use `region` if you want to distinguish resources by region

Reply:
Right. Will correct it right away